2. Services/2. Services.md

```diff
@@ -22,21 +22,21 @@ Let's talk about these categories.
 
 #### 2.0.0.0 Validations
 
-Validations focus on ensuring that the incoming or outgoing data match a particular set of rules, which can be structural, logical or external validations, in that exact order of priority. We will go in details about this in the upcoming sections.
+Validations ensure that incoming or outgoing data match a particular set of rules: structural, logical, or external validations, in that exact order of priority. We will go into details about this in the upcoming sections.
 
 #### 2.0.0.1 Processing
 
-Processing mainly focuses on the flow-control, mapping and computation to satisfy a business need - the processing operations specifically is what distinguishes one service from another, and in general one software from another.
+Processing mainly focuses on the flow-control, mapping, and computation to satisfy a business need - the processing operations distinguish one service from another and, in general, one software from another.
 
 #### 2.0.0.2 Integration
 
-Finally, the integration process is mainly focused on retrieving or pushing data from or to any integrated system dependencies.
+Finally, the integration process focuses on retrieving or pushing data from or to any integrated system dependencies.
 
-Every one of these aspects will be discussed in details in the upcoming chapter, but the main thing that should be understood about services is that they should be built with the intent to be pluggable and configurable so they are easily integrated with any technology from a dependency standpoint and also be easily plugged into any exposure functionality from an API perspective.
+We will discuss these aspects in detail in the upcoming chapter. The main thing to understand about services is to design to be pluggable and configurable, so they are easily integrated with any technology from a dependency standpoint and easily plugged into any exposure functionality from an API perspective.
```

Comment:
Something is not right with this sentence @dlandi.
``` The main thing to understand about services is to design to be pluggable and configurable```

Reply:
@hassanhabib  I fixed the phrasing. Thanks.